source/common/network/apple_dns_impl.cc

```diff
@@ -44,16 +44,21 @@ DNSServiceErrorType DnsService::dnsServiceGetAddrInfo(DNSServiceRef* sdRef, DNSS
   return DNSServiceGetAddrInfo(sdRef, flags, interfaceIndex, protocol, hostname, callBack, context);
 }
 
-AppleDnsResolverImpl::AppleDnsResolverImpl(Event::Dispatcher& dispatcher)
-    : dispatcher_(dispatcher) {
+// Parameters of the jittered backoff strategy.
+static constexpr uint32_t RetryInitialDelayMilliseconds = 300;
```

Comment:
use std::chrono types

Reply:
even though they are passed on to the backoff constructor as uint?

Reply:
That seems broken also. I would fix that as well.

Reply:
Yeah, I can totally fix the backoff classes if you'd like to see that. I'd rather do it in a separate PR so that I can fix uses unrelated to this change.

Reply:
Yes that's fine but I would still make these constants chronos